src/main/kotlin/bsu/cc/parser/XlsxParser.kt

```diff
@@ -149,3 +146,31 @@ private fun rowToCellMap(row: Row): Map<Int, Cell> {
 
     return output
 }
+
+fun highlightRow(
+        workbook: Workbook,
+        sheetIndex: Int,
+        rowIndex: Int,
+        color: IndexedColors,
+        extendToMatchHeader: Boolean = false
+) {
+    val sheet = workbook.getSheetAt(sheetIndex) ?: throw IllegalArgumentException("Sheet does not exist at given index")
+    val highlightStyle = workbook.createCellStyle()
```

Comment:
Interesting, this seems to imply that the workbook stores a list of styles that cells are mapped to instead of just storing the style information directly with the cell.